test/functional/ui/screen_basic_spec.lua

```diff
@@ -79,6 +87,31 @@ describe('Screen', function()
     end)
   end)
 
+  describe('startup', function()
+    it('default startup screen', function()
```

Comment:
Yes, I should do that. This is a test for the behavior the 2nd commit should fix, it makes sure the laststatus value is applied.